graphsdk/src/androidTest/java/com/microsoft/graph/http/DefaultConnectionFactoryTests.java

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.graph.http;
+
+import android.test.AndroidTestCase;
+
+import java.io.IOException;
+
+/**
+ * Test cases for {@see DefaultConnectionFactory}
+ */
+public class DefaultConnectionFactoryTests extends AndroidTestCase {
+
+    public void testCreateFromRequest() {
```

Comment:
trim down this method to something prettier by like: assertNotNull(new DefaultConnectionFactory().createFromRequest(nwe MockHttpRequest()));